pkg/moov/transfer_api.go

```diff
@@ -31,10 +32,36 @@ func WithTransferIdempotencyKey(key uuid.UUID) CreateTransferArgs {
 	}
 }
 
+type CreateTransferTimeoutError struct {
+	TransferStarted TransferStarted
+}
+
+func (e CreateTransferTimeoutError) Error() string {
```

Comment:
This just means they need to convert the error every time to determine if they need to reverse the transfer where as before we can just return three arguments. Completed, ongoing, and failed

Reply:
If it's expected that callers dig into TransferStarted all the time our godocs need to mention that.